All_Relationships/7.prior_probabilities_calculation.py

```diff
@@ -0,0 +1,149 @@
+
+# coding: utf-8
+
+# # Calculate Prior Probability of Edge Types
+
+# This notebook calculates the prior probabiltity of an edge type through permutation. This notebook will be used for various edge types as this project progresses.
+
+# In[ ]:
+
+
+from collections import defaultdict
+import itertools
+import statistics
+import pandas as pd
+
+import tqdm
+from hetio.permute import permute_pair_list
+get_ipython().magic(u'matplotlib inline')
+
+
+# In[ ]:
+
+
+hetnet_df = pd.read_csv('hetnet_dg_kb.csv')
+
+
+# In[ ]:
+
+
+disease_degree = dict(hetnet_df["disease_id"].value_counts())
+gene_degree = dict(hetnet_df["gene_id"].value_counts())
+
+
+# In[ ]:
+
+
+association_edge = defaultdict(set)
+association_row = list()
```

Comment:
I do think you can be more clear by using plural for list variable names when appropriate. For example, any reason not to call this `association_rows`?